docs/visualize/time-series-visual-builder.asciidoc

```diff
@@ -65,6 +65,15 @@ template syntax to customize the Markdown with data based on a set of series.
 image:images/tsvb-markdown.png["Markdown Visualization"]
 
 
+==== Table
+
+The table visualization allows you to present data from multiple series in one
+table. You can define a grouping field to define rows in your table and add multiple
+columns via series to the table.
+
+image:images/tsvb-table.png["Table Visualization"]
+
```

Comment:
How about:

A table visualization allows you to display data from multiple time series. You define which field group to show in the rows and what columns of data to display.